package.json

```diff
@@ -28,30 +28,30 @@
   },
   "homepage": "https://github.com/coders51/rabbitmq-stream-js-client#readme",
   "devDependencies": {
-    "@tsconfig/node16": "^1.0.3",
+    "@tsconfig/node-lts": "^20.1.1",
```

Comment:
Isn't it deprecated?

Reply:
...is it? I couldn't find anything on google, do you happen to have some source I can look into?